### DISCUSSION TITLE
How do I change the upower icon size?

### DISCUSSION THREAD

**Discussion by bsvh** [JakeStanger/ironbar#158] [Q&A] [answered]
I'm using the upower module to display battery status, but battery icon is way bigger than any other icon. It seems to take up all the vertical space in the panel. It there a way to adjust it's size?

**Comment by yavko** [comment]
Does the icon size setting not exist for it?

**Comment by bsvh** [comment]
What do you mean by icon size setting? I don't see anything for that documented. The only thing relevant that I can see is the `.upower .icon` css selector, but I haven't been able to get that to change the size.

**Reply by JakeStanger** [reply]
Using `.upower .icon { size: 10px }` (or whatever size) in your CSS should work. If that doesn't, I'd consider that a bug.

**Reply by bsvh** [reply]
That's not working for me. I'm getting `'size' is not a valid property name` error and the css isn't loading. Should I put in an issue for it then?

**Reply by yavko** [reply]
> That's not working for me. I'm getting `'size' is not a valid property name` error and the css isn't loading. Should I put in an issue for it then?

I think it's different for icons in GTK css, check gtk3 css property page

**Reply by bsvh** [reply, marked as answer]
> I think it's different for icons in GTK css, check gtk3 css property page

Thanks for the hint. I was able to do it using `.upower .icon { -gtk-icon-transform: scale(0.5); }`

**Reply by JakeStanger** [reply]
Ah sorry, I meant font-size, but sounds like I was wrong anyway. I didn't know about gtk-icon-transform. Glad you got it sorted & thanks for helping Yavko

**Reply by yavko** [reply]
> Ah sorry, I meant font-size, but sounds like I was wrong anyway. I didn't know about gtk-icon-transform. Glad you got it sorted & thanks for helping Yavko

Glad to have been useful 🙂
